### ISSUE TITLE
If PhoneticProperties data is included, NoMethodError (undefined method `+' for nil:NilClas) will be raised.

### ISSUE THREAD

**Issue by sinoue-1003** [martijn/xsv#27]
Asian strings such as Japanese can contain phonetic string data (furigana/yomigana)
In this case, the data structure will be as follows, and SharedStringsParser will not initialize `@current_string`, which will cause an error.

In order to deal with this, I would like to add a process to skip when an rPh is received.
Can I create a PR for this?

### exapmle
``` phonetic.xml
<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<sst xmlns="http://schemas.openxmlformats.org/spreadsheetml/2006/main" count="35" uniqueCount="33">
<rPh><t>phonetic</t></rPh>
<si><t>Some strings</t></si>
```

### error
```
irb(main):006:0>     @file = File.open("test/files/phonetic.xml")
irb(main):007:0>     strings = Xsv::SharedStringsParser.parse(@file)
Traceback (most recent call last):
        7: from bin/console:14:in `<main>'
        6: from (irb):7
        5: from /xsv/lib/xsv/shared_strings_parser.rb:9:in `parse'
        4: from /xsv/lib/xsv/sax_parser.rb:19:in `parse'
        3: from /xsv/lib/xsv/sax_parser.rb:19:in `loop'
        2: from //xsv/lib/xsv/sax_parser.rb:43:in `block in parse'
        1: from /xsv/lib/xsv/shared_strings_parser.rb:33:in `characters'
NoMethodError (undefined method `+' for nil:NilClass)
```

**Comment by martijn** [comment]
Pull requests are welcome!

In the SheetRowsParser rPh elements are simply ignored (5a24108), but I had not seen a file yet with phonetic data in the shared strings.

If you can please add an example file and a test case `test/shared_strings_parser_test.rb` to prevent future regressions.

Let me know if you need any help!

**Comment by sinoue-1003** [comment]
Thanks for sharing the information.
Now, I'll add the test and create the PR.

**Comment by martijn** [comment]
Fixed in Xsv 1.0.2
